processor/metricstransformprocessor/README.md

```diff
@@ -311,3 +311,29 @@ operations:
   action: group
   group_resource_labels: {"resouce.type": "container", "source": "kubelet"}
 ```
+
+### Attributes Processor for Metrics vs. Metric Transform Processor
+
+Regarding metric support, these two processors have overlapping functionality. They can both do simple modifications
```

Comment:
I think a lint to the Attributes Processor would be useful here

Reply:
I'll add a link